react/features/filmstrip/components/Filmstrip.web.js

```diff
@@ -107,7 +110,12 @@ class Filmstrip extends Component {
                         className = 'filmstrip__videos'
                         id = 'filmstripLocalVideo'
                         onMouseOut = { this._onMouseOut }
-                        onMouseOver = { this._onMouseOver } />
+                        onMouseOver = { this._onMouseOver }>
+                        {
+                            interfaceConfig.filmStripOnly
+                                ? null : <InviteButton />
+                        }
+                    </div>
```

Comment:
I'm feeling like this could be a prop instead, be it a new one or an existing one. I think this.props.displayToolbox is already passed in for filmstrip only too. Reducing the usage of the global interfaceConfig might be warranted in general when configurations can be passed in, that way component assume less about the world around them and become more encapsulated.

Reply:
I'll check the displayToolbox prop. I also considered a prop, but my thinking was that once we put interfaceConfig in the store it will be easy to replace all calls. However, it will also mean that most of our components are redux aware even when probably not needed, so yeah I'll see if I can replace it with a prop.